src/rpc/names.cpp

```diff
@@ -1054,10 +1055,12 @@ name_checkdb (const JSONRPCRequest& request)
   ).Check (request);
 
   ChainstateManager& chainman = EnsureChainman (request.context);
+  NodeContext& node = EnsureNodeContext(request.context);
```

Comment:
Tiny nit:  Please add a space before the function-call parenthesis to match the style in this file.